tests/monotouch-test/CoreGraphics/PDFContextTest.cs

```diff
@@ -76,6 +76,7 @@ public void Context_Url_Rect_Info ()
 		}
 
 		[Test]
+		[Ignore ("Crash on simulator using xcode 9.3 beta 1")]
 		public void Constructors ()
 		{
 			Assert.Throws<Exception> (() => new CGContextPDF ((CGDataConsumer) null), "null CGDataConsumer");
```

Comment:
Should we have a bug/issue about this so it is not forgotten in a future beta?

Reply:
good idea, I wanted to do a rdar after investigating them but let's also create an issue to track PR with (hopefully) temporary hacks